.changeset/empty-pants-live.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wpengine/wp-graphql-content-blocks": patch
+---
+
+tests : Core list blocks and core list-item blocks
```

Comment:
Always use an imperative tense for these commit logs (same as PR titles) per conventional commits.

I.e it should fill in the blank for the following sentence: `If merged, this PR will ___________.`
```suggestion
tests : Add tests for `CoreList` and `CoreListItem` blocks.
```

Reply:
Done 👍  Thankyou for this trick : " `If merged, this PR will ___________.` ", now i will always keep this in mind while framing commit messages